test/runify.js

```diff
@@ -442,8 +442,56 @@ hosts.forEach(function (record) {
         }).then(record => {
           const result = record[0];
           assert(!result.stdout.match(/2/), 'Unexpected stdout: ' + result.stdout);
-        })
-      })
+        });
+      });
+
+      it('creates "optional" environments correctly (hostArgs)', function() {
+        // browsers are irrelevant to this test
+        //
+        if (['firefox', 'chrome', 'remote'].includes(type)) {
+          this.skip();
+          return;
+        }
+
+        let source = '';
+        let hostArguments = '';
+
+        // Setup special cases
+        if (type === 'ch') {
+          hostArguments = '-Intl-';
+          source = 'print(typeof Intl === "undefined");';
+        }
```

Comment:
This is presently failing in CI because the pre-compiled binary that we're using to run these tests is a Release build and does not expose optional flags. I'm in the process of setting up an S3 bucket (with a little help from my friends) to host our own pre-compiled binaries, allowing us a location for the latest and all future Debug builds